.github/workflows/ios-unsigned.yml

```
@@ -0,0 +1,83 @@
+          zip -r ../${XCODE_SCHEME}.ipa Payload
+          cd ..
+
+      - name: Package .app and logs
+        run: |
+          cd $ARCHIVE_PATH/Products/Applications
+          zip -r ../../$OUTPUT_DIR/${XCODE_SCHEME}_app.zip "${XCODE_SCHEME}.app"
+          cd ../..
+          mv build-log.txt $OUTPUT_DIR/
+
+      - name: Upload build artifacts
+        uses: actions/upload-artifact@v3
```

actionlint (1.7.7)

the runner of "actions/upload-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)